wireguard/wireguard.go

```diff
@@ -1490,7 +1490,7 @@ func (w *Wireguard) ensureLinkAddressV4(netlinkClient netlinkshim.Interface) err
 		}
 		logCxt.Debug("Address set on wireguard device")
 	}
-
+	logCxt.Debug("Address not set on wireguard device")
```

Comment:
This log will fire even if the "Address set" log fires

Reply:
good catch, rookie mistake on my part :laughing: